src/actuator/ActuatorFunctorsSimple.C

```diff
@@ -0,0 +1,487 @@
+
+  alpha = alphaNoTwist + twist;  
+}
+
+
+void 
+AirfoilTheory2D::calculate_cl_cd(
+    double alpha,
+    std::vector<double> aoatable,
+    std::vector<double> cltable,
+    std::vector<double> cdtable,
+    double &cl,
+    double &cd)
+{
+  // Get cl and cd from the tables
+  utils::linear_interp(aoatable, cltable, alpha, cl);
+  utils::linear_interp(aoatable, cdtable, alpha, cd);
+
+  // Do another other processing needed on cl/cd
+  // [..nothing at this time..]
+}
+
+ActSimpleSetUpThrustCalc::ActSimpleSetUpThrustCalc(ActuatorBulkSimple& actBulk)
+  : actBulk_(actBulk)
+{
+}
+
+void
+ActSimpleSetUpThrustCalc::operator()(int index) const
+{
+  auto hubLoc = Kokkos::subview(actBulk_.hubLocations_, index, Kokkos::ALL);
+  auto hubOri = Kokkos::subview(actBulk_.hubOrientation_, index, Kokkos::ALL);
+  auto thrust = Kokkos::subview(actBulk_.turbineThrust_, index, Kokkos::ALL);
+  auto torque = Kokkos::subview(actBulk_.turbineTorque_, index, Kokkos::ALL);
+
+  for (int i = 0; i < 3; i++) {
+    thrust(i) = 0.0;
+    torque(i) = 0.0;
+  }
+
+  for (int j = 0; j < 3; j++) {
+    hubLoc(j) = 0.0;
+    hubOri(j) = 0.0;
+  }
+}
+
+void
+ActSimpleComputeThrustInnerLoop::operator()(
+  const uint64_t pointId,
+  const double* nodeCoords,
+  double* sourceTerm,
+  const double dual_vol,
+  const double scvIp) const
+{
+
+  auto offsets = actBulk_.turbIdOffset_.view_host();
+
+  // shouldn't thrust and torque contribs only come from blades?
+  // probably not worth worrying about since this is just a debug calculation
+
+  // determine turbine
+  /*
+  int turbId = 0;
+  const int nPointId = static_cast<int>(pointId);
+  for (; turbId < offsets.extent_int(0); turbId++) {
+    if (nPointId >= offsets(turbId)) {
+      break;
+    }
+  }
+  */
+  if (NaluEnv::self().parallel_rank()<actBulk_.num_blades_) {
+    int turbId = NaluEnv::self().parallel_rank();
+  //auto hubLoc = Kokkos::subview(actBulk_.hubLocations_, turbId, Kokkos::ALL);
+  //auto hubOri = Kokkos::subview(actBulk_.hubOrientation_, turbId, Kokkos::ALL);
+  auto thrust = Kokkos::subview(actBulk_.turbineThrust_, turbId, Kokkos::ALL);
+  auto torque = Kokkos::subview(actBulk_.turbineTorque_, turbId, Kokkos::ALL);
+
+  double r[3], rPerpShaft[3], forceTerm[3];
+
+  for (int i = 0; i < 3; i++) {
+    //forceTerm[i] = sourceTerm[i] * scvIp;
+    forceTerm[i] = sourceTerm[i]*scvIp;
+    //r[i] = nodeCoords[i] - hubLoc(i);
+    thrust(i) += forceTerm[i];
+  }
+
+  torque(0) = 0.0;
+  torque(1) = 0.0;
+  torque(2) = 0.0;
+  }
+}
+
+ActSimpleStashOrientationVectors::ActSimpleStashOrientationVectors(
+  ActuatorBulkSimple& actBulk)
+  : orientation_(helper_.get_local_view(actBulk.orientationTensor_)),
+    offset_(helper_.get_local_view(actBulk.turbIdOffset_)),
+    turbId_(actBulk.localTurbineId_)
+{
+  helper_.touch_dual_view(actBulk.orientationTensor_);
+  actBulk.turbIdOffset_.sync_host();
+}
+
+void
+ActSimpleStashOrientationVectors::operator()(int index) const
```

Comment:
Looks like this doesn't really do anything either so just delete it.

Reply:
Yup, should be deleted.